docs/sources/panels-visualizations/query-transform-data/transform-data/index.md

```diff
@@ -154,7 +154,7 @@ Use this transformation to add a new field calculated from two other fields. Eac
 - **Alias** - (Optional) Enter the name of your new field. If you leave this blank, then the field will be named to match the calculation.
 - **Replace all fields** - (Optional) Select this option if you want to hide all other fields and display only your calculated field in the visualization.
 
-> **Note:** **Cumulative functions** and **Window functions** modes are experimental features. Engineering and on-call support is not available. Documentation is either limited or not provided outside of code comments. No SLA is provided. Enable the **addFieldFromCalculationStatFunctions** feature toggle in Grafana to use this feature. Contact Grafana Support to enable this feature in Grafana Cloud.
```

Comment:
I think this PR contains the [work](https://github.com/grafana/grafana/pull/79495) of the window/cumulative stuff also. So maybe the other can be closed? Or remove that work from here to keep them fully separated? I wouldn't have a problem with just keeping it here and rename the PR.

Reply:
Oh, must have committed the docs changes on the wrong branch. I’ll clean this up

Reply:
@oscarkilhed it's also the feature toggle stuff, not just the docs. There's both 'addFieldFromCalculationStatFunctions' and 'formatString' work in this PR and then there's this other [PR](https://github.com/grafana/grafana/pull/79495) with only 'addFieldFromCalculationStatFunctions'. I should've been more specific

Reply:
Oh, thank you for catching that! I guess I based this branch off of the cumulative/window branch.